sites/all/modules/custom/lesson_batch/lesson_batch.pages.inc

```diff
@@ -0,0 +1,140 @@
+<?php
+/**
+ * @file
+ * Renders output data.
+ */
+
+/**
+ * Page callback: Displays a a table with uploaded files.
+ *
+ * @see lesson_batch_menu().
+ */
+function lesson_batch_landing_table() {
+  drupal_add_js(drupal_get_path('module', 'lesson_batch') . '/js/lesson_batch.js');
+  drupal_add_js(array('uid' => $GLOBALS['user']->uid, ), 'setting');
+  $uploaded_files = [];
+  $uploaded_files = db_select('file_managed', 'fm')
+    ->fields('fm', ['fid', 'uid', 'uri', 'status'])
+    ->execute()
+    ->fetchAll();
+  $user_has_rights_to_download = FALSE;
+  foreach ($uploaded_files as $file) {
+    if ($file->uid === $GLOBALS['user']->uid) {
+      $user_has_rights_to_download = TRUE;
+      break;
```

Comment:
Code styles

Reply:
Sorry, you mean that variable name is a little weird?